.github/workflows/go.yml

```diff
@@ -8,6 +8,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
+    services:
+      devnet:
+        image: shardlabs/starknet-devnet:0.2.4
```

Comment:
What's your opinion on pulling latest here instead of v0.2.4?

Reply:
I have fixed